src/sentry/data_secrecy/api/waive_data_secrecy.py

```diff
@@ -79,7 +79,7 @@ def get(self, request: Request, organization: Organization) -> Response:
             )
         except Http404:
             return Response(
-                {"detail": "No data secrecy waiver in place."}, status=status.HTTP_404_NOT_FOUND
+                {"detail": "No data secrecy waiver in place."}, status=status.HTTP_200_OK
```

Comment:
can you not allow 404 in useApiQuery? you are looking for something that doesn't exist here

Reply:
ill hendle this in the frontend